TACC/Core-Styles
# feat: no macos overscroll behavior

## Overview

Do not allow macOS overscroll behavior.

Lead designer (M.S.) does not want that behavior on any of our sites.

## Related

- supports 

## Changes

- **added** css to prevent browser default "overscroll" behavior on body

## Testing

Already tested on TUP. Works consistently on any page when using macOS.

0. Use a browser on an Apple device.
1. Add this new style to body (live HTML edit is fine):

    ```css
    body { overscroll-behavior-y: none }
    ```
2. Verify weird scrolling stops.

## UI

### Before

https://github.com/TACC/Core-Styles/assets/62723358/1db77d1e-58c6-4ded-a8a2-36af832fccd5

### After

https://github.com/TACC/Core-Styles/assets/62723358/7bc00a40-88ed-4a38-a2db-f1b83e0c2ad4
